### ISSUE TITLE
Gjs-WARNING **: JS ERROR: TypeError: texture_cache.load_icon_name is not a function

### ISSUE THREAD

**Issue by genodeftest** [sustmi/gnome-shell-extensions-sustmi#27]
Hi
I just ran into this which causes logging by gnome-shell:
```
Gjs-WARNING **: JS ERROR: TypeError: texture_cache.load_icon_name is not a function
enable/updatePositions@/home/christian/.local/share/gnome-shell/extensions/windowoverlay-icons@sustmidown.centrum.cz/extension.js:167
enable/wsWinOverInjections.relayout<@/home/christian/.local/share/gnome-shell/extensions/windowoverlay-icons@sustmidown.centrum.cz/extension.js:217
injectToFunction/parent[name]@/home/christian/.local/share/gnome-shell/extensions/windowoverlay-icons@sustmidown.centrum.cz/extension.js:53
Workspace<._doAddWindow@resource:///org/gnome/shell/ui/workspace.js:1530
wrapper@resource:///org/gnome/gjs/modules/lang.js:169
Workspace<._windowAdded@resource:///org/gnome/shell/ui/workspace.js:1538
wrapper@resource:///org/gnome/gjs/modules/lang.js:169
WorkspaceThumbnail<.acceptDropInternal@resource:///org/gnome/shell/ui/workspaceThumbnail.js:589
wrapper@resource:///org/gnome/gjs/modules/lang.js:169
ThumbnailsBox<.acceptDrop@resource:///org/gnome/shell/ui/workspaceThumbnail.js:791
wrapper@resource:///org/gnome/gjs/modules/lang.js:169
_Draggable<._dragActorDropped@resource:///org/gnome/shell/ui/dnd.js:449
wrapper@resource:///org/gnome/gjs/modules/lang.js:169
_Draggable<._onEvent@resource:///org/gnome/shell/ui/dnd.js:163
wrapper@resource:///org/gnome/gjs/modules/lang.js:169
_getEventHandlerActor/<@resource:///org/gnome/shell/ui/dnd.js:54
```
Running gnome-shell 3.14.3

**Comment by sustmi** [comment]
Hi Christian, thanks for reporting.
It looks like the `load_icon_name()` function was removed in Gnome Shell 3.7.
I didn't noticed the bug because it occurs only if the extension tries to load default icon (for applications that do not have any icon).
It is fixed in 4b319c169f41e2c00416da9c5987af97da1d9d51 and should propagate to extensions.gnome.org soon. (https://extensions.gnome.org/extension/302/windowoverlay-icons/ - version 15)

**Comment by genodeftest** [comment]
Thanks for the fix, works fine!
